src/WebJobs.Script/OutOfProc/Http/DefaultHttpInvokerService.cs

```diff
@@ -2,24 +2,147 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
+using System.Net.Http.Formatting;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.WebApiCompatShim;
+using Microsoft.AspNetCore.WebUtilities;
 using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
 
-namespace Microsoft.Azure.WebJobs.Script.OutOfProc
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
 {
     public class DefaultHttpInvokerService : IHttpInvokerService
     {
         private readonly HttpClient _httpClient;
+        private readonly string _httpInvokerBaseUrl;
 
         public DefaultHttpInvokerService(HttpClient httpClient)
         {
             _httpClient = httpClient;
+            // TODO: user agent ok?
+            _httpClient.DefaultRequestHeaders.UserAgent.ParseAdd(HttpInvokerConstants.UserAgentHeaderValue);
+            _httpInvokerBaseUrl = "http://localhost:8090/";
         }
 
-        public Task GetInvocationResponse(ScriptInvocationContext scriptInvocationContext)
+        public async Task InvokeAsync(ScriptInvocationContext scriptInvocationContext)
         {
-            throw new NotImplementedException();
+            var requestUri = _httpInvokerBaseUrl + scriptInvocationContext.FunctionMetadata.Name;
+            if (Utility.IsSimpleHttpTriggerFunction(scriptInvocationContext.FunctionMetadata))
+            {
+                await ProcessSimpleHttpInvocationRequest(scriptInvocationContext, requestUri);
+                return;
+            }
+            await ProcessDefaultInvocationRequest(scriptInvocationContext, requestUri);
+        }
+
+        private async Task ProcessSimpleHttpInvocationRequest(ScriptInvocationContext scriptInvocationContext, string requestUri)
+        {
+            HttpRequestMessage httpRequestMessage = null;
```

Comment:
@paulbatum - Added code to detect if invocation request is for a simple http triggered function and forwards invocation request to the HttpInvoker and response from the HttpInvoker is set at invocation response for the function. 
Please see corresponding example HttpInvoker here:  https://github.com/pragnagopa/functions-http-worker/blob/88416d8c1065796bc3de95810e80b14ac768162f/go/src/httpServer/helloworldServer.go#L87-L104

Reply:
Ahh, this is super nice, thank you.